docs/reference/compose_kill.md

```diff
@@ -15,13 +15,12 @@ $ docker compose kill -s SIGINT
 | `--remove-orphans` | `bool`   |           | Remove containers for services not defined in the Compose file |
 | `-s`, `--signal`   | `string` | `SIGKILL` | SIGNAL to send to the container                                |
 
-
 <!---MARKER_GEN_END-->
 
 ## Description
 
 Forces running containers to stop by sending a `SIGKILL` signal. Optionally the signal can be passed, for example:
 
 ```console
-$ docker compose kill -s SIGINT
+docker compose kill -s SIGINT
 ```
```

Comment:
I think for our docs, we still include a prompt in these cases to distinguish process output from command that's executed. When using the `copy` button on the examples, the prompt is stripped so that it's not on the user's clipboard; e.g. see https://docs.docker.com/reference/cli/docker/compose/kill/